code/part-one/tests/01-Signing.js

```diff
@@ -28,7 +32,29 @@ describe('Signing module', function() {
   });
 
   describe('sign', function() {
+    const message = randomBytes(16);
+    let publicKey = null;
+    let signature = null;
+
+    beforeEach(function() {
+      const privateKey = signing.createPrivateKey();
+      publicKey = signing.getPublicKey(privateKey);
+      signature = signing.sign(privateKey, message);
+    });
+
+    it('should return a hex string', function() {
+      expect(signature).to.be.a('string').and.match(/^[0-9a-f]*$/);
```

Comment:
Can we do `expect(signature).to.be.a.hexString;` like at https://github.com/hyperledger/education-cryptomoji/blob/4f082ac694eb9953ee2d27cf0aad02af5f66e75c/code/part-two/client/tests/01-Signing.js#L23?